__snapshots__/index.test.js.snap

```diff
@@ -2237,7 +2237,16 @@ exports[`should pass on packages/eslint-config-react/test/samples/my-component/M
 
 exports[`should pass on packages/eslint-config-react/test/samples/my-component/MyComponent.old.js 1`] = `Array []`;
 
-exports[`should pass on packages/eslint-config-react/test/samples/my-component/MyComponent.test.js 1`] = `Array []`;
+exports[`should pass on packages/eslint-config-react/test/samples/my-component/MyComponent.test.js 1`] = `
+Array [
+  Object {
+    "column": 43,
+    "line": 4,
+    "rule": "react/jsx-handler-names",
```

Comment:
This error shouldn't happen as the function is anonymous..  I guess that's a deal breaker for now, see: https://github.com/yannickcr/eslint-plugin-react/issues/2586